XROLE/storeManus_v2
# CH webpack setup #162778270

#### What does this pull request do?
- Set up webpack alongside babel
#### Description of task to be completed?
- [x] Install Webpack and it dependencies
- [x] Install babel and it's depencies
- [x] Configure babel to transpile es6+ and JSX
- [x] Configure webpack to build react scripts

#### How should this be manually tested?
- Clone the repo, change directory into storeManus_v2
- Run npm install and then npm start. You will see a simple html page with just a button served by the configured webpack.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#162778270]
